### ISSUE TITLE
cmake error

### ISSUE THREAD

**Issue by boyob** [weixu000/libtorch-yolov3-deepsort#8]

**Comment by mathpopo** [comment]
(base) chenxin@Nitro-AN515:~/Downloads/libtorch-yolov3-deepsort/build$ make
Scanning dependencies of target detection
[  5%] Building CXX object detection/CMakeFiles/detection.dir/src/Darknet.cpp.o
/home/chenxin/Downloads/libtorch-yolov3-deepsort/detection/src/Darknet.cpp: In member function ‘void Detector::Darknet::create_modules()’:
/home/chenxin/Downloads/libtorch-yolov3-deepsort/detection/src/Darknet.cpp:192:17: error: ‘BatchNorm’ is not a member of ‘torch::nn’
                 torch::nn::BatchNorm bn = torch::nn::BatchNorm(bn_options(filters));
                 ^~~~~
/home/chenxin/Downloads/libtorch-yolov3-deepsort/detection/src/Darknet.cpp:193:35: error: ‘bn’ was not declared in this scope
                 module->push_back(bn);
                                   ^~
detection/CMakeFiles/detection.dir/build.make:81: recipe for target 'detection/CMakeFiles/detection.dir/src/Darknet.cpp.o' failed
make[2]: *** [detection/CMakeFiles/detection.dir/src/Darknet.cpp.o] Error 1
CMakeFiles/Makefile2:175: recipe for target 'detection/CMakeFiles/detection.dir/all' failed
make[1]: *** [detection/CMakeFiles/detection.dir/all] Error 2
Makefile:102: recipe for target 'all' failed
make: *** [all] Error 2

**Comment by mathpopo** [comment]
must be pytorch 0.4?

**Comment by mathpopo** [comment]
i use 1.9

**Comment by mathpopo** [comment]
i modify to 1.0.0 
-- Added CUDA NVCC flags for: -gencode;arch=compute_61,code=sm_61
CMake Error at libtorch/share/cmake/Caffe2/public/utils.cmake:16 (add_library):
  add_library cannot create target "caffe2_library" because another target
  with the same name already exists.  The existing target is an interface
  library created in source directory
  "/home/chenxin/Downloads/libtorch-yolov3-deepsort/detection".  See
  documentation for policy CMP0002 for more details.
Call Stack (most recent call first):
  libtorch/share/cmake/Caffe2/Caffe2Config.cmake:121 (caffe2_interface_library)
  libtorch/share/cmake/Torch/TorchConfig.cmake:39 (find_package)
  tracking/CMakeLists.txt:2 (find_package)


CMake Error at libtorch/share/cmake/Caffe2/public/utils.cmake:16 (add_library):
  add_library cannot create target "caffe2_gpu_library" because another
  target with the same name already exists.  The existing target is an
  interface library created in source directory
  "/home/chenxin/Downloads/libtorch-yolov3-deepsort/detection".  See
  documentation for policy CMP0002 for more details.
Call Stack (most recent call first):
  libtorch/share/cmake/Caffe2/Caffe2Config.cmake:123 (caffe2_interface_library)
  libtorch/share/cmake/Torch/TorchConfig.cmake:39 (find_package)
  tracking/CMakeLists.txt:2 (find_package)
